test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -645,7 +645,12 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 			if concreteOp.Namespace != nil {
 				namespace = *concreteOp.Namespace
 			} else {
+				// define Pod's namespace automatically, and create that namespace.
 				namespace = fmt.Sprintf("namespace-%d", opIndex)
+				_, err := client.CoreV1().Namespaces().Create(ctx, &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}, metav1.CreateOptions{})
```

Comment:
Should we create namespace every time since this is creating pods, what if the namespace already exists?

Reply:
Hm, you are right. 
In most cases, namespaces created here shouldn't exist because they use `opIndex` in their name.
But, we have `createNamespacesOp` and if a user creates a namespace named `namespace-{number}` with that, it may introduce error because of `already exist`.

I changed it not to exec `b.Fatalf` if a namespace is already exist.